Vagrantfile

```diff
@@ -263,10 +269,10 @@ def install_borg(fuse)
     pip install -U wheel  # upgrade wheel, too old for 3.5
     cd borg
     # clean up (wrong/outdated) stuff we likely got via rsync:
-    rm -f borg/*.so borg/*.cpy*
-    rm -f borg/{chunker,crypto,compress,hashindex,platform_linux}.c
-    rm -rf borg/__pycache__ borg/support/__pycache__ borg/testsuite/__pycache__
+    rm -rf __pycache__
+    find src -name '__pycache__' -exec rm -rf {} \;
```

Comment:
this did not match the src/borg/... dir structure any more and thus frequently causes troubles.
the rm and find are both needed, did not want to start the find in "." as it then also finds/removes all `__pycache__` in .tox.